JhipsterXamarin/JhipsterXamarin/JhipsterXamarin/Services/AuthenticationService.cs

```diff
@@ -35,12 +35,14 @@ public async Task<bool> SignIn(LoginModel loginModel)
             return IsAuthenticated;
         }
 
-        public void SignOut()
+        public string SignOut()
         {
-            _httpClient.DefaultRequestHeaders.Remove(AuthorizationHeader);
-            JwtToken = null;
-            IsAuthenticated = false;
-            CurrentUser = null;
+            string mess = null;
+                _httpClient.DefaultRequestHeaders.Remove(AuthorizationHeader);
+                JwtToken = null;
+                IsAuthenticated = false;
+                CurrentUser = null;
+            return mess;
```

Comment:
Here, mess is always null, what's the point ?

Reply:
I just wanted a nullable string as the SignOut's result, we can change it to add something or change it as a boolean or an integer.